### PR TITLE
support groovy to reconfigure all release parts

### PR DESCRIPTION
**Pull request by dirk-thomas**

### CODE REVIEW

File: ros_buildfarm/jenkins.py [hunk 1]
```diff
@@ -100,6 +100,8 @@ def configure_view(
         template_name='generic_view.xml.em', dry_run=False):
     view_config = get_view_config(
         template_name, view_name, include_regex=include_regex)
+    if not jenkins:
```
Comment (tfoote) [inline ros_buildfarm/jenkins.py]: Why do you return the config object here?

Comment (dirk-thomas) [inline ros_buildfarm/jenkins.py]: Because it is needed outside to generate the groovy script.

Comment (tfoote) [inline ros_buildfarm/jenkins.py]: Ok, it's surprising to have it return either a `view_config`, `None`, or a `view`

**Comment by tfoote** [comment]
+1

**Comment by dirk-thomas** [comment]
Regression with non-release reconfigure jobs fixed in b491c74ca03fb0b4949174b0c19960fbd6ec721d.

**Pull request merged**